lib/rubocop/cop/mixin/multiline_literal_brace_layout.rb

```diff
@@ -3,21 +3,29 @@
 
 module RuboCop
   module Cop
-    # Common functionality for checking that the closing brace of a literal is
-    # symmetrical with respect to the opening brace and contained elements.
+    # Common functionality for checking the closing brace of a literal is
+    # either on the last line as the contained elements, or a new line.
```

Comment:
`last line as the contained elements` -> `same line as the last contained element`